frontend/src/components/Reports/Routine.js

```diff
@@ -63,21 +52,15 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByTest",
+          link:"/ByTestType?type=indicator&report=activityReportByTest",
           label: <FormattedMessage id="sideNav.label.bytesttype"/>,
         },
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByPanel",
+          link:"/ByPanelType?type=indicator&report=activityReportByPanel",
```

Comment:
can we rename the path to `/ActivityReport?type=indicator&report=activityReportByPanel`